index.html

```diff
@@ -57,50 +81,35 @@ <h3>Use Cases Discussed</h3>
           <li>A web browser in private browsing mode</li>
         </ul>
       </section>
-      <section class="informative">
-        <h3>Definitions Discussed</h3>
-        <p>
-          A specification's definition of a webview should not refer to specific technologies like Android WebView or WKWebView. The general consensus appeared to be the definition should focus on how the web content is used. Definitions discussed included:
-        </p>
-        <ul>
-          <li>Whether the user can enter their own URL to navigate to</li>
-          <li>
-            "Embedded" (e.g. WKWebView) vs "System" (e.g. Custom Tabs) WebViews, which differ by whether they run in a separate context, or the same context as a web browser on the device
-          </li>
-          <li>Whether the user may "freely navigate the web" (i.e. browsing, since they are called "web browsers" after all)</li>
-          <li>Whether certain UX features are available (e.g. managing security and privacy, back/forward button, bookmarks...)</li>
-        </ul>
-      </section>
-      <section class="informative">
+      <section >
         <h3>Additional Points Raised</h3>
         <ul>
-          <li>In-app browsers like SFSafariViewController are usually virtually identical to a regular web browser, differing only by user interface</li>
+          <li>In-app browsers like SFSafariViewController are usually virtually identical to a regular web browser, differing only by user interface.</li>
         </ul>
       </section>
-      <section class="informative">
-        <h3>Proposed Definition</h3>
+    </section>
+    <section id="definition">
+      <h2>Proposed Definition</h2>
+      <section>
+        <h3>What is a WebView?</h3>
```

Comment:
There might be two ways to define a WebView

1. WebView is a framework / component
2. A WebView can or cannot do certain things other user agents do